content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md

```diff
@@ -120,33 +118,44 @@ sure that its parent commit is pushed to Git.
 
 </admon>
 
-Updates to the live metrics are highlighted in orange in the
+Updates to the live metrics are highlighted (in orange) in the project table and
 [compare pane](/doc/studio/user-guide/projects-and-experiments/visualize-and-compare#compare-experiments)
-as well.
+in real time.
 
-The number of experiments with new updates to the live metrics values are
-displayed in the `Live` icon, which can also be used to filter and show only
-running experiments in the table.
+The number of experiments with recent updates to live metrics are displayed in
+the `Live` icon, which can also be used to filter and show only running
+experiments in the table.
 
 Live plots are displayed in the
-[plots pane](/doc/studio/user-guide/projects-and-experiments/visualize-and-compare#how-to-generate-plots);
-you can see the plots getting populated as Studio receives new updates.
+[plots pane](/doc/studio/user-guide/projects-and-experiments/visualize-and-compare#how-to-generate-plots).
+You can see them getting populated as Studio receives new updates.
 
 ![](https://static.iterative.ai/img/studio/live_plots.gif)
 
 An experiment can have one of the following statuses:
 
 - **Running** - Iterative Studio expects to receive live metrics and plots for
-  these experiments. Note that if the experiment stops due to any error,
-  Iterative Studio will not be aware of this and it will continue to wait for
-  live updates. In this case, you should delete the live metrics row from the
-  project table in Iterative Studio.
+  these experiments.
+
+  <admon type="warn">
+
+  Note that if the experiment stops due to any error, Iterative Studio will not
+  be aware of this and it will continue to wait for live updates. In this case,
```

Comment:
There's no timeout for Studio to consider an exp failed? Like, 24 hours or something?

Reply:
We don't atm. Right @mvshmakov? We should though.

Reply:
No, I don't think we have such a timeout. Created https://github.com/iterative/studio/issues/5138 to address. Meanwhile, deleting the row is a good enough suggestion IMO.